packages/neutrino/bin/start.js

```diff
@@ -2,16 +2,33 @@ const ora = require('ora');
 const debounce = require('lodash.debounce');
 const { start } = require('../src');
 const base = require('./base');
+const DashboardPlugin = require("webpack-dashboard/plugin");
+const { spawn } = require('child_process');
 
 module.exports = (middleware, args) => {
   const spinner = ora({ text: 'Building project' });
+  const argsClone = Object.assign({}, args);
+
+  if (argsClone.dashboard) {
+    argsClone.quiet = true;
+
+    middleware.push(neutrino => {
+      neutrino.config.plugin('dashboard').use(DashboardPlugin);
+    });
+
+    const child = spawn(`${__dirname  }/../../../node_modules/.bin/webpack-dashboard`, [], { stdio: 'inherit' });
+
+    process.on('exit', () => {
+      child.kill();
+    })
+  }
 
   return base({
     middleware,
-    args,
+    argsClone,
```

Comment:
These are named args, so if we do decide we need a clone of `args`, we need to pass this as:

```js
args: argsClone,
```